task-json/q2.js

```diff
@@ -0,0 +1,8 @@
+const timeinseconds = [4,5,2,10,0];
+timeinseconds.forEach(function  (element){
+    setTimeout(() => {
+        console.log(`print ${element} after ${element} seceonds`);
+
+    },element*1000);
+   

```

Comment:
now the code is missy, please note the following:
- create a function called "delay" and take one parameter called seconds
- use a promise to handle the setTimeout
- the numbers will not be printed in the same order, the item (2) will be printed before (5) , what I want is (wait 4 seconds, then print, then wait 5 seconds then ...)